cloud_governance/main/environment_variables.py

```diff
@@ -48,7 +52,7 @@ def __init__(self):
             self._environment_variables_dict['account'] = self.get_aws_account_alias_name().upper().replace('OPENSHIFT-', '')
         self._environment_variables_dict['policy'] = EnvironmentVariables.get_env('policy', '')
 
-        self._environment_variables_dict['aws_non_cluster_policies'] = ['ec2_idle', 'ec2_stop', 'ec2_run', 'ebs_in_use',
+        self._environment_variables_dict['aws_non_cluster_policies'] = ['ec2_idle', 'ec2_stop', 'ebs_in_use',
```

Comment:
Why do u remove ec2_run from this list ?

Reply:
It is the static approach. It is very bad to have like this.
Look this new [ADR Dir structure](https://docs.google.com/document/d/1hPv80Xe6-Cao59smi3Yougsa9lfhYparpxVTmNpIM94/edit). I added **policy/cleanup**, so all cleanup policies will reside in this folder and we will walk through this and run the policies.